webapp/app/static/css/base.css

```diff
@@ -37,6 +37,11 @@ body {
     font-size: var(--text-medium);
 }
 
+/* SPACING */
+.spacing-t-07 {
+    padding-top: var(--spacing-07);
+}
+
```

Comment:
Thought re: naming: the rest of the CSS classes seem fairly semantic, may be more consistent to make this a class specifically for YesNoFields, e.g. `yes-no-field`.